app/controllers/users/passwords_controller.rb

```diff
@@ -25,6 +26,12 @@ def update
 
     private
 
+    def pii_requested_but_locked?
```

Comment:
let's rename this to be a statement not a question? Since it has side effects, maybe `capture_password_if_pii_requested_but_locked`

Reply:
done!